docs/internet-identity-spec.adoc

```diff
@@ -536,6 +540,23 @@ This flow is the boring default
 +
 (This could include a button to go to the management screen, or maybe this _is_ the management screen with a info box.)
 
+=== Flow: Setup recovery
+1. The user is offered two options for recovery
+  * A security key
+  * A BIP-39 seed phrase
+  * (Skip?)
+2. Depending on their choice
+ a) If they choose the seed phrase it is generated and displayed to them with a copy button and an explanation of having to keep this phrase secure
+ b) 👆 If they choose the security key they generate new credentials for the touched device
```

Comment:
Do we prevent using the same security here (using `excludeCredentials`)? Or is that future work?

Reply:
We do now. I've implemented it both here and for the `Add device` button on the management page.